dvc/utils/fs.py

```diff
@@ -20,15 +20,21 @@ def get_inode(path):
     return inode
 
 
-def get_mtime_and_size(path):
-    stat = os.stat(path)
-    size = stat.st_size
-    mtime = stat.st_mtime
+def get_mtime_signature_and_size(path, ignore_file_handler=None):
```

Comment:
This reads as "mtime, signature and size" and one is surprised at the call site why it only returns 2 values, not 3.

Reply:
So I suggest either keep the old name or invent something better.